Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -102,14 +99,14 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
 	UFUNCTION(BlueprintCallable, Category = "C++")
 	void UnlockAllLevels();
 
-	/** Is called to update the stars actors and in widgets */
-	UFUNCTION(BlueprintCallable, Category = "C++")
-	void RefreshProgressionUIElements();
-
 	/** Returns difficultyMultiplier */
 	UFUNCTION(BlueprintCallable, Category="C++")
 	float GetDifficultyMultiplier();
 
+	/** Returns spot component by the player tag, returns null if spot is not found */
+	UFUNCTION(BlueprintCallable, Category="C++")
```

Comment:
Please remember add `BlueprintPure` for Getter functions, to generate this function in blueprint without exec in\out:
```
	UFUNCTION(BlueprintCallable, BlueprintPure, Category="C++")
```

Reply:
done https://github.com/h4rdmol/ProgressionSystem/pull/40/commits/3e6d9bd7dfeed8016b25220faffa4960021062d3